apps/expo/src/components/FilledButtons.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Text, TouchableOpacity, View } from "react-native";
+
+interface FilledButtonProps {
+  onClick: () => void;
+  disabled?: boolean;
+  children: React.ReactNode;
+}
+
+const FilledButton: React.FC<FilledButtonProps> = ({
+  onClick,
+  disabled = false,
+  children,
+}) => {
+  if (disabled) {
+    return (
+      <TouchableOpacity onPress={onClick} disabled={disabled}>
+        <View className="inline-flex h-12 items-center justify-center gap-2">
```

Comment:
The button shouldn't have any css that affects how its parent might use it -- for the screenshot you sent, it should be on the left of the screen directly under the text if there is no styling in the parent view

Reply:
okay sounds good mb!